xray/cli.go

```diff
@@ -310,6 +312,15 @@ func scanCmd(c *cli.Context) error {
 	return commands.Exec(scanCmd)
 }
 
+func addTrailingSlashToRepoPathIfNeeded(c *cli.Context) string {
+	repoPath := c.String("repo-path")
+	if repoPath != "" && !strings.Contains(repoPath, "/") {
+		// In case a only repo name was given (no path) we are adding a trailing slash.
```

Comment:
In case a only repo name was given... --> In case only repo name was provided...